docs/studies-cs.md

```diff
@@ -0,0 +1,62 @@
+---
+id: studies-cs
+title: CONVALESCENT SERUM (CS) STUDY
+hide_title: true
+hide_table_of_contents: true
+---
+
+##### Top
+
+<h1>Convalescent <span>Serum</span></h1>
+
+**RESEARCH QUESTION: For patients with new COVID-19 infection, how does the addition of convalescent serum (CS) affect the outcomes (1) severe disease (indicated by mechanical ventilation) and (2) inpatient death?**
+
+### Background
+
+Convalescent Serum (CS) is currently under study as possible treatments for COVID-19. Before randomized clinical trials read out, a retrospective observational study can provide insight into the efficacy of this treatment.
+
+### Study Design
+
+The study examines the effect of CS treatment in hospitalized COVID-19 patients on disease severity. The study involves two study groups (those receiving or not receiving CS), as depicted below:
+
+![Convalescent Serum cohort pathways](cs_cohorts.png)
+
+Four outcomes are considered:
```

Comment:
We actually aren't stratifying the outcomes this way. We're looking at ventilation separately from death, so there really are only two outcomes.

Reply:
ok, fixed.